plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -340,6 +340,65 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         }
     }
 
+    if(flb_sds_cmp(ctx->resource, "generic_node",
+                    flb_sds_len(ctx->resource)) == 0 ||
+        flb_sds_cmp(ctx->resource, "generic_task",
+                    flb_sds_len(ctx->resource)) == 0) {
+
+        ctx->is_generic_resource_type = FLB_TRUE;
+
+        tmp = flb_output_get_property("location", ins);
```

Comment:
[Optional] In the past we've discussed specifying labels as a map, so instead of hard-coding every resource type, we can just extract the resource and a freeform set of labels from the config. Not proposing that we do it now, but that may be a good thing to keep in mind for the next version bump…

Reply:
That is a specific PR I have for the future :-) It will be more involved than these are right now, so I decided to separate the task of doing that into its own since the previous one where I tried to do all the things together turned into a bit of a mess.